custom-steps/expat/post-build.ps1

```diff
@@ -1,13 +0,0 @@
-param (
-    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
-)
-
-Import-Module "$PSScriptRoot/../../ps-modules/Build" -DisableNameChecking -Force
-
-if ((Get-IsOnMacOS)) {
-    Remove-DylibSymlinks -BuildArtifactsPath $BuildArtifactsPath
-}
```

Comment:
I think we want to keep this step for Mac.  On Mac, they create a symlink to a symlink to the actual dylib in this build.  This cleans it up so it only needs the actual dylibs.

We also don't properly handle publishing symlinks in these builds, so it produces 3 separate copies of the .dylib file without this step:

![image](https://github.com/TechSmith/ThirdParty-Packages-vcpkg/assets/62569405/4f9acc98-b2fc-4844-aab4-bcf4ab025a9f)